packages/test/test-end-to-end-tests/src/test/sharedStringEndToEndTests.spec.ts

```diff
@@ -15,16 +15,23 @@ import {
     ChannelFactoryRegistry,
     ITestFluidObject,
 } from "@fluidframework/test-utils";
-import { describeFullCompat } from "@fluidframework/test-version-utils";
+import { describeNoCompat } from "@fluidframework/test-version-utils";
+import { AttributorProvider } from "@fluid-internal/attributor";
 
 const stringId = "sharedStringKey";
 const registry: ChannelFactoryRegistry = [[stringId, SharedString.getFactory()]];
 const testContainerConfig: ITestContainerConfig = {
     fluidDataObjectType: DataObjectFactoryType.Test,
     registry,
+    loaderProps: {
+        scope: { IAttributorProvider: new AttributorProvider() },
+        options: {
+            trackAttribution: true
+        }
+    }
 };
 
-describeFullCompat("SharedString", (getTestObjectProvider) => {
+describeNoCompat("SharedString", (getTestObjectProvider) => {
```

Comment:
hint: `describeFullCompat.noCompat` will do the same thing, and doesn't require an import change. just a quicker way to do what you did.